deploy/operator.yaml

```diff
@@ -22,12 +22,13 @@ spec:
           imagePullPolicy: Always
           env:
             - name: WATCH_NAMESPACE
```

Comment:
Do we need to add `value: ""`?

https://github.com/operator-framework/operator-sdk/blob/master/doc/operator-scope.md#cluster-scoped-operator-usage